tests/unit/mathematics/linalg/DenseMatrixExactLog_unittest.cc

```diff
@@ -43,54 +36,27 @@ TEST(DenseMatrixExactLog, dense_log_det)
 	CDenseMatrixOperator<float64_t>* op=new CDenseMatrixOperator<float64_t>(mat);
 	SG_REF(op);
 
-	// create operator function with the operator and the engine to submit jobs
-	CDenseMatrixExactLog *op_func=new CDenseMatrixExactLog(op, e);
+	// create operator function with the operator
+	CDenseMatrixExactLog* op_func = new CDenseMatrixExactLog(op);
 	SG_REF(op_func);
 
 	// its really important we call the precompute on the operato function
 	op_func->precompute();
 
-	// for storing the aggregators that submit_jobs return
-	CDynamicObjectArray aggregators;
+	float64_t result = 0.0;
 
 	// create samples for extracting the trace of log(C) and submit
 	for (index_t i=0; i<size; ++i)
```

Comment:
maybe this can be parallell as well?
or is it inside the other parallel loop?

Reply:
this is the other loop and i think we can keep it as is maybe ?